tools/PTE/CITest/scenarios/FAB-17350.sh

```diff
@@ -0,0 +1,208 @@
+#!/bin/bash
+
+#
+# Copyright IBM Corp. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# testcase: FAB-17350
+# 1 channel, 1 org, 1 peer, 1 ca, solo orderer
+# multiple processes (4, 8, 12, ..., 52) X 10000 transactions, both invoke and query
+
+kubeCfg=""
+channelSetup=""
+
+TESTCASE="FAB-17350"
+NREQ=10000
+myKey0=0
+
+CWD=$PWD
+CurrentDirectory=$(cd `dirname $0` && pwd)
+FabricTestDir="$(echo $CurrentDirectory | awk -F'/fabric-test/' '{print $1}')/fabric-test"
+#operator dir
+OPDir=$FabricTestDir"/tools/operator"
+
+pteReport=$OPDir"/pteReport.txt"
+
+pteInputMaster="testdata/barebones-test-input-master.yml"
+pteInput="testdata/barebones-test-input.yml"
+
+usage () {
+    echo -e "\nUSAGE:\t./FAB-1730.sh [options] [values]"
+    echo
+    echo -e "-h\tview this help message"
+
+    echo -e "-c\tbring up network, create/join channel, install/instantiate chaincode"
+    echo -e "\t\t(optional, Default: no)"
+
+    echo -e "-k\tkubenetes configuration file with relative path to fabric-test/tools/operator/"
+    echo -e "\t\ta local network will be brought up if this option is not present"
+
+    echo
+    echo -e "example:"
+    echo -e "\tThe following command will bring up a network on kubenetes based on the specified kube config file, then create/join channel, install/instantiate chaincode, and send transactions."
+    echo -e "\t./FAB-17350.sh -c -k launcher/FAB-17350-kubeConfig/kube-config-wdc04-community-migration.yml"
+    echo
+    echo -e "\tThe following command will send transactions to the network on kubenetes specified in the specified Kube configuration file."
+    echo -e "\t./FAB-17350.sh -k launcher/FAB-17350-kubeConfig/kube-config-wdc04-community-migration.yml"
+    echo
+    echo -e "\tThe following command will bring up a local network, then create/join channel, install/instantiate chaincode, and send transactions."
+    echo -e "\t./FAB-17350.sh -c"
+    echo
+    echo -e "\tThe following command will send transactions to the local network."
+    echo -e "\t./FAB-17350.sh"
+
+    exit
+}
+
+networkProc() {
+    cd $OPDir
+    if [ "$kubeCfg" != "" ]; then
+        # bring down network
+        echo "[$0] bring down the kube network"
+        echo "go run main.go -i testdata/barebones-networkspec-spec.yaml -k $kubeCfg -a down"
```

Comment:
that would obviate these echo statements

Reply:
use -z to check if a variable exist and use set -x instead